Bottom: 739a68b4f57f0f0eb2ed2dff067ebc5d7d38c39d
Top:    4fa2d2eb3eeeff4c1dc33c67228929fcfd37b5cf
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-08-02 14:52:02 -0700

Refresh of rs-setroute

---

diff --git a/src/rsocket.c b/src/rsocket.c
index e3406a4..d64c3bb 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1250,7 +1250,7 @@ resolve_route:
 					      RDMA_OPTION_IB_PATH, rs->optval,
 					      rs->optlen);
 			free(rs->optval);
-			rs->optval = 0;
+			rs->optval = NULL;
 		} else {
 			ret = rdma_resolve_route(rs->cm_id, to);
 		}
